sdk/go/pulumi-language-go/main.go

```diff
@@ -135,18 +137,50 @@ func (host *goLanguageHost) Run(ctx context.Context, req *pulumirpc.RunRequest)
 		return nil, errors.Wrap(err, "failed to prepare environment")
 	}
 
+	// by default we try to run a named executable on the path, but we will fallback to 'go run' style execution
+	goRunInvoke := false
+
 	// The program to execute is simply the name of the project.  This ensures good Go toolability, whereby
 	// you can simply run `go install .` to build a Pulumi program prior to running it, among other benefits.
 	program, err := findProgram(req.GetProject())
 	if err != nil {
-		return nil, errors.Wrap(err, "problem executing program (could not run language executor)")
+		const message = "problem executing program (could not run language executor)"
```

Comment:
Just `message :=`.

Reply:
You and @pgavlin are [in opposition here](https://github.com/pulumi/pulumi/pull/3503#discussion_r345994737)